maintainers/scripts/haskell/update-hackage.sh

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env nix-shell
-#! nix-shell -i bash -p nix curl jq nix-prefetch-github -I nixpkgs=.
+#! nix-shell -i bash -p nix curl jq nix-prefetch-github git gnused -I nixpkgs=.
```

Comment:
This change seems to be left over.

Reply:
What do you mean. It’s a correct fix. It’s just not super on topic for this PR. Should I put it into it’s own commit?

Reply:
nvm